modules/defer-applicaton-tuning-example.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="defer-application-of-tuning-changes_{context}"]
+= Deferring application of tuning changes
+
+Administrators use the Node Tuning Operator (NTO) to update custom resources (CRs) on a running system and make tuning changes. For example, they can update or add a sysctl parameter to the [sysctl] section of the tuned object. When administrators apply a tuning change, the NTO prompts TuneD to reprocess all configurations, causing the tuned process to roll back all tuning and then reapply it.
+
+Latency-sensitive applications might not tolerate this removal and reapplication of the tuned profile. Configurations that partition CPUs and control process/interrupt affinity through the performance profile are particularly affected. To mitigate this issue, {product-title} introduced new methods for applying tuning changes. Before {product-title} 4.17, the only available method, immediate, applied changes instantly, often triggering a tuned restart.
+
+The key additional methods supported are `always` and `update`;
+ 
+ * `always` means every change is applied at the next node reboot.
+ * `update` means when a tuning change causes a tuned profile to change, it is treated as `immediate`. When a tuning change does not cause a tuned profile to change and its values are modified in place, it is treated as `always`.
```

Comment:
I think `immediate` should not highlighted, because this implies it has a special meaning. It kinda refers one of the oldest name we had earlier in the development. 
I'd write something lie

`update` means when a tuning change causes a tuned profile to change, it is treated as immediate, so applied with the default treatment, as soon as possible. When a tuning change does not cause a tuned profile to change and its values are modified in place, it is treated as `always`.